setup.py

```diff
@@ -5,31 +5,44 @@
 from setuptools import setup
 from setuptools.command.install import install
 
-FULLVERSION = '0.0.2-2'
+FULLVERSION = "0.0.3"
 VERSION = FULLVERSION
 
 with open(os.path.join(os.path.dirname(__file__), "README.md")) as infile:
     LONG_DESCRIPTION = infile.read()
 
-setup(name='xdem',
-      version=FULLVERSION,
-      description='Set of tools to manipulate Digital Elevation Models (DEMs) ',
-      long_description=LONG_DESCRIPTION,
-      long_description_content_type="text/markdown",
-      url='https://github.com/GlacioHack/xdem',
-      author='The GlacioHack Team',
-      author_email="this-is-not-an-email@a.com",  # This is needed for PyPi unfortunately.
-      license='BSD-3',
-      packages=['xdem'],
-      install_requires=[
-          'numpy', 'scipy', 'rasterio', 'geopandas',
-          'pyproj', 'tqdm', 'scikit-gstat', 'scikit-image',
-          "geoutils @ https://github.com/GlacioHack/geoutils/tarball/main"
-      ],
-      extras_require={'rioxarray': ['rioxarray'], 'richdem': ['richdem'], 'pdal': [
-          'pdal'], 'opencv': ['opencv'], "pytransform3d": ["pytransform3d"]},
-      scripts=[],
-      zip_safe=False)
+setup(
+    name="xdem",
+    version=FULLVERSION,
+    description="Set of tools to manipulate Digital Elevation Models (DEMs) ",
+    long_description=LONG_DESCRIPTION,
+    long_description_content_type="text/markdown",
+    url="https://github.com/GlacioHack/xdem",
+    author="The GlacioHack Team",
+    author_email="this-is-not-an-email@a.com",  # This is needed for PyPi unfortunately.
+    license="BSD-3",
+    packages=["xdem"],
+    install_requires=[
+        "numpy",
+        "scipy",
+        "rasterio",
+        "geopandas",
+        "pyproj",
+        "tqdm",
+        "scikit-gstat",
```

Comment:
Ideally this would also be an optional dependency as most users will be not necessarily be interested in the spstats part... I think it adds up a lot of additional dependencies no?

Reply:
In its current state, the spstats dependencies are required. We could discuss in the future if that should be the case (@rhugonnet), but for now, this is how it should be!